src/updates/node/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as install from './installer';
```

Comment:
I think you're fine building everything into this `index.ts` file, the other products are split out as we're dealing with different parts of the products (i.e appc cli core and appc cli installer)